import "ydb/library/actors/protos/actors.proto";

package NKikimrColumnShardBlobOperationsProto;

message TEvDeleteSharedBlobs {
    optional uint64 SourceTabletId = 1;
    optional NActorsProto.TActorId SourceActorId = 2;
    optional string StorageId = 3;
    repeated string BlobIds = 4;
}

message TEvDeleteSharedBlobsFinished {
}
